test/unittest/meson.build

```diff
@@ -10,4 +10,13 @@ unittest_nntrainer_internal = executable('unittest_nntrainer_internal',
   install_dir: application_install_dir
 )
 
+
+unittest_util_func = executable('unittest_util_func',
+  'unittest_util_func.cpp',
+  dependencies: [unittest_nntrainer_internal_deps],
+  install: get_option('enable-test'),
+  install_dir: application_install_dir
+)
+
 test('unittest_nntrainer_internal', unittest_nntrainer_internal)
+test('unittest_util_func', unittest_util_func)
```

Comment:
how about moving `test(...)` to the below of each executable?

Reply:
Fixed :)